docs/workload.rst

```diff
@@ -0,0 +1,193 @@
+VM Workloads Search
+==============
+
+Visualize the inventory of vSphere workloads that do not have Carbon Black Cloud sensors installed.
+
+Fetch Compute Resource by ID
+----------------------------
+
+Using a query of the ``ComputeResource`` object, you can get the compute resource by ID from your organization.
+
+::
+
+    >>> from cbc_sdk import CBCloudAPI
+    >>> from cbc_sdk.workload import ComputeResource
+
+    >>> # This is an example id that we want to query
+    >>> id = 15054425
+
+    >>> cbc = CBCloudAPI()
+    >>> query = cbc.select(ComputeResource, id)
+
+    >>> # A string object is returned here, so we can print the result directly.
+    >>> print(query)
+
+    ComputeResource object, bound to https://defense-dev01.cbdtest.io.
+     Last refreshed at Mon Mar  1 12:02:14 2021
+    -------------------------------------------------------------------------------
+
+              appliance_uuid: c89f183b-f201-4bca-bacc-80184b5b8823
+                cluster_name: example-cluster-name
+                  created_at: 2020-11-18T07:41:16.834Z
+             datacenter_name: None
+                 eligibility: NOT_ELIGIBLE
+            eligibility_code: ['Launcher not found']
+               esx_host_name: 10.105.7.129
+               esx_host_uuid: bb8d2842-0438-9a74-7964-1d0efad10f28
+                   host_name: localhost.localdomain
+                          id: 15054425
+         installation_status: NOT_INSTALLED
+        installation_status_code: None
+                  ip_address: 10.105.7.201
+                        name: CB-ServiceTest
+             os_architecture: 64
+              os_description: CentOS 7 (64-bit)
+                     os_type: CENTOS
+                        uuid: 5022227f-947a-84f8-5816-747f5e18e5ac
+            vcenter_host_url: 10.105.5.63
+                vcenter_name: VMware vCenter Server 7.0.0 build-15952599
+                vcenter_uuid: 4a6b1382-f917-4e1a-8564-374cb7274bd7
+         vmwaretools_version: 10336
+
+
+And there we have it, the Carbon Black Cloud Python SDK makes querying as easy as writing just a few lines of code.
```

Comment:
Probably don't need this line of text. I usually try to avoid telling users something is easy, because if they don't think it's easy, it makes them mad

Reply:
lol, seems legit :D